frontend/src/lib/components/widgets/DateInput/DateInput.test.tsx

```diff
@@ -32,6 +32,7 @@ const getProps = (elementProps: Partial<DateInputProto> = {}): Props => ({
     label: "Label",
     default: ["1970/01/01"],
     min: "1970/1/1",
+    format: "YYYY/MM/DD",
```

Comment:
Nit: can we use a different day or month value to disambiguate these test dates (e.g. 1970/1/20) so that it's clear at a glance whether the day/month has been accidentally transposed?

Reply:
Good point, changed it 👍